libdrgn/configure.ac

```diff
@@ -56,6 +56,9 @@ dnl We need dwarf_cu_getdwarf() which was added in elfutils 0.160.
 PKG_CHECK_MODULES(libdw, [libdw >= 0.160])
 AC_SUBST([libdw_CFLAGS])
 AC_SUBST([libdw_LIBS])
+PKG_CHECK_MODULES(libkdumpfile, [libkdumpfile])
```

Comment:
Since libkdumpfile doesn't appear to be packaged for most distros, let's make this dependency optional.

Reply:
I did not forget about this. Coming from illumos, I'm new to autotools+pkgconf so this may take me some time.

Reply:
Just pushed the work to ensure that this is an optional dependency. This is my first attempt at doing anything like this so please do not hesitate to ask for corrections or nitpick.